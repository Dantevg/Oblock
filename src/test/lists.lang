{
	printList = list => print(...list)
	
	a = [10, x = 20, 30]
	b = [...a, ...a]
	
	print(a.2)
	print(a.x)
	print(...a)
	print b
}
