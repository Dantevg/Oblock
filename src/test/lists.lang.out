30
20
10	30
[10, 30, 10, 30]
